### DISCUSSION TITLE
Time picker with seconds

### DISCUSSION THREAD

**Discussion by jtklein** [web-ridge/react-native-paper-dates#340] [Show and tell] [answered]
Hello,
in a fork of this repository, I added the possibility to also select seconds from the time picker.
Is that of general interest to make a pull request for this addition?

**Comment by iM-GeeKy** [comment, marked as answer]
Hi @jtklein, I'll let @RichardLindhout be the final decision on this, but since this feature doesn't adhere to the [Time Picker](https://m3.material.io/components/time-pickers/overview) in Material Design, my guess is we'd respectfully pass on this pull request. However, if you have solved any other open issues, we'd be happy to merge them!
